css/project.less

```diff
@@ -21,27 +21,31 @@
     transition: width .3s ease-in-out .3s, border-width .3s ease-in-out .3s, padding .3s linear .3s;
   }
 
+  &._resize {
```

Comment:
Yep no worries, will change.
PS here it is not for private/public separation, I usually use `_` prefixes in css for classes that represent states, e.g. `_active`, `_open`, etc. But definitely better to be consistent, so will rename to `resizing`